ddl/util/util.go

```diff
@@ -204,6 +212,78 @@ func LoadGlobalVars(ctx context.Context, sctx sessionctx.Context, varNames []str
 	return nil
 }
 
+const (
+	newSessionRetryInterval = 200 * time.Millisecond
+	logIntervalCnt          = int(3 * time.Second / newSessionRetryInterval)
+
+	// NewSessionDefaultRetryCnt is the default retry times when create new session.
+	NewSessionDefaultRetryCnt = 3
+	// NewSessionRetryUnlimited is the unlimited retry times when create new session.
+	NewSessionRetryUnlimited = math.MaxInt64
+)
+
+// NewSession creates a new etcd session.
+func NewSession(ctx context.Context, logPrefix string, etcdCli *clientv3.Client, retryCnt, ttl int) (*concurrency.Session, error) {
```

Comment:
Why did we put this function here? This function is not necessarily related to DDL, as it is also used in "info.go"?